### ISSUE TITLE
Add option for scaling the reference to match the distorted resolution

### ISSUE THREAD

**Issue by stderr-to-devnull** [fifonik/FFMetrics#150]
Like many other people, I scale down then encode. In this scenario I am interested in `VMAF` score result in a comparison between a **_scaled down_** reference, not the original resolution because that loss of detail is expected and a conscientious choice.

So it is obvious that the VMAF score will be very skewed in this case. The current workflow is fine and has its merits, I am asking for an option to take the loss of detail due to resolution downscaling out of the equation when calculating the quality metrics (I am personally looking at `VMAF` but it will obviously apply to all).

TL;DR: a checkmark that says `scale down reference to match distorted resolution`

**Comment by fifonik** [comment]
About your example (scaling down + multi encoding):
- It does not matter what metric value you got at the end. You only need to know what value is the best so there is no reason to compate with "downscaled reference". The only possible benefit I see -- speed (as smaller frame size processed facter).
- If you want to compare with "downscaled reference", you have to use the same scaling method as the one you used while downscaled + encoded. Other reference downscaling method can introduce different artefacts so the comparison will make no sense.

So I believe that reference must be kept with no changes.

As a result, your suggestion does not looks right to me.
Sorry.
